NOTICE

```diff
@@ -85,3 +85,8 @@ go-zookeeper - Native ZooKeeper client for Go
 https://github.com/samuel/go-zookeeper
 Copyright (c) 2013, Samuel Stauffer <samuel@descolada.com>
 See https://github.com/samuel/go-zookeeper/blob/master/LICENSE for license details.
+
+We also include code from a variety of npm packages. For details, see
```

Comment:
Waiting on further input from @swinslow here on how to properly attribute npm package dependencies.